src/CommunityToolkit.Maui/Views/Popup/PopupContainer.cs

```diff
@@ -0,0 +1,69 @@
+using CommunityToolkit.Maui.Core;
+using Microsoft.Maui.Controls.PlatformConfiguration;
+using Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific;
+
+namespace CommunityToolkit.Maui.Views;
+
+partial class PopupContainer<T> : PopupContainer
+{
+	readonly TaskCompletionSource<PopupResult<T>> taskCompletionSource;
+
+	public PopupContainer(Popup<T> content, TaskCompletionSource<PopupResult<T>> taskCompletionSource) :base (content, null)
```

Comment:
shouldn't we add argument validation for `taskCompletionSource`? (content will be validated on base ctor)